app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="container">
+  <div class="child mt-4">
+    <div class="mb-3">
+      <%= button_to "Back", authenticated_root_path, method: :get, class: 'btn btn-outline-primary'%>
+    </div>
+    <div class="text-center">
+      <h2>Add a New Transaction</h2>
+    </div>
+      <%= form_with model: @expense, url: group_expenses_path(params[:group_id]) do |form| %>
+        <div class="mb-3">
+          <%= form.label :name %>
+          <%= form.text_field :name, placeholder:'Name Here!', class:'text-input form-control form-control-lg' %>
+        </div>
+        <div class="mb-3">
+          <%= form.label :amount %>
+          <%= form.number_field :amount, placeholder:'Amount Here!', class:'text-input form-control form-control-lg' %>
+        </div>
+        <div class="input-group mb-3">
+          <%= form.label :group_id %>
+          <%= form.select :group_id, options_from_collection_for_select(@groups, :id, :name) %>
+        </div>
+        <div class="mb-3">
+          <%= form.submit "Save", class: "btn btn-primary" %>
+        </div>
+      <% end %>
+  </div>
+</div>
```

Comment:
- [OPTIONAL] From the design template, the form pages for _**adding a new category and adding a new transaction**_  should have a navigation bar, with the `background-color` of `#3778c2` and an icon to go back to the previous page. Also, notice the structure of the form inputs used and the colours that were used on the page. It would be nice for you to match your design to that of the design template provided for this project. 👍 😎 

| Form Page - Category | Form Page - Transactions | Design Template |
| ------------------------ | --------------------------- | -------------------|
| ![image](https://user-images.githubusercontent.com/77106962/236014236-aa70aa23-0141-4856-a1c9-14160cfcae8d.png) | ![image](https://user-images.githubusercontent.com/77106962/236014385-77736aea-449d-41db-969e-4d3ae7d1be02.png)| ![image](https://user-images.githubusercontent.com/77106962/236007469-b8ea948e-aa46-4bee-8659-89450c16bfdf.png) |